pkg/cli/init.go

```diff
@@ -45,8 +45,8 @@ func (c *cli) newInitCmd() *cobra.Command {
 	// so that it shows up in help and does not cause a parse error.
 	cmd.Flags().String(projectVersionFlag, c.defaultProjectVersion,
 		fmt.Sprintf("project version, possible values: (%s)", strings.Join(c.getAvailableProjectVersions(), ", ")))
-	// The --plugins flag can only be called to init projects v2+.
-	if c.projectVersion != config.Version1 {
+	// The --plugins flag can only be called to init projects v3+.
+	if c.projectVersion != config.Version2 {
```

Comment:
Should this conditional continue to check `c.projectVersion != config.Version1`? Or in the case `v1` is used, explicitly print out a message saying that version is now deprecated? We could use the `IsVersionSupported()` helper to accomplish this.

Reply:
V1 is no longer supported and its code no longer exists in the project.

Reply:
i agree that it needs to be IsVersionSupported()

Reply:
If we do this change here it then, shows that it will not work with v3+